__tests__/genaralTestConfi.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+import { MONGO_URI, MONGO_OPTIONS } from '../src/config';
+
+export const mongoseConnect = async (User) => {
+  const connect = await mongoose.connect(MONGO_URI, MONGO_OPTIONS, (err) => {
+    if (err) {
+      process.exit(1);
+    }
+  });
+  await User.deleteMany({ email: 'team04@buidsdg.com' });
+  return connect;
+};
+
+export const mongoseDisconnect = async () => {
+  const disconnect = await mongoose.connection.close();
+  return disconnect;
+};
+
+export const userData = {
+  email: 'team04@buidsdg.com',
```

Comment:
I am sure this email is dummy, but shouldn't it read 'team042@buildsdg.com'

Reply:
will be fix in next PR